### DISCUSSION TITLE
[Projet Beta] Repository not available

### DISCUSSION THREAD

**Discussion by silopolis** [community/community#16031] [Projects and Issues]
Hi folks,

Giving a try at Projects Beta to see if it can help us manage some of our work, namely documentation work on LinuxCNC...

I have a fork at https://github.com/silopolis/linuxcnc, created a project that I seem to have associated with it (https://github.com/silopolis/linuxcnc/projects?type=beta).

I started by dumping my paper todo as draft issues... But then, when I wanted to create actual issues I discovered that my linuxcnc repository wasn't available in the list :'(

I tried to see if I did something wrong or if there was a clue in my repository settings, without success.

Is there something forbidding me to manage projects on this repo ? something I don't understand ?

Any help welcome

TY
J

**Comment by MathieuDR** [comment]
I'm having somewhat the same issue. As a collaberator of a private repo I cannot change my draft to issues on said repo.

**Reply by silopolis** [reply]
@MathieuDR just a quick one to tell you I just opened a support ticket for this problem. I linked this discussion in the ticket but will let you know what comes out of it anyway...

**Comment by silopolis** [comment]
hi Mathieu,

Le ven. 20 mai 2022 à 12:25, Mathieu ***@***.***> a écrit :
>
> I'm having somewhat the same issue. As a collaberator of a private repo I
cannot change my draft to issues on said repo.

At least I'm not alone ! After so much time since I posted this here and in
forum (
https://github.community/t/projet-beta-repository-not-available/248827)
without any echo, I was really starting to wonder if I was asking something
dumbly stupid ? 😬

I've tried to check project publicity and other settings to no luck :/

Let's hope we'll be heard someday...

**Comment by zicklag** [comment]
Yeah, for some reason it seems like certain repositories just won't pop up in the list when I try to add issues to it.

It seems like some of my forks pop up and things, like that, but for some reason not all of them.

**Comment by zicklag** [comment]
Hey, wait, I just discovered, you can paste the URL of an issue to add it to the board!
